orchagent/vxlanorch.cpp

```diff
@@ -350,6 +409,15 @@ bool VxlanTunnel::createTunnel(MAP_T encap, MAP_T decap)
 
         ids_.tunnel_id = create_tunnel(ids_.tunnel_encap_id, ids_.tunnel_decap_id, ip, gUnderlayIfId);
 
+        shared_ptr<DBConnector> m_counter_db = shared_ptr<DBConnector>(new DBConnector(COUNTERS_DB, DBConnector::DEFAULT_UNIXSOCKET, 0));
```

Comment:
Why add to counter table? Can you capture this in Design Doc if it is needed

Reply:
In doc/L2-vxlan-hld.md, please see chapter 1.6.1, 'Add the tunnel name map to counter table, so that the 'show mac' command can display the FDB learnt from VXLAN tunnel.'.